core/dropdowndiv.js

```diff
@@ -163,7 +160,8 @@ DropDownDiv.positionToField_ = null;
  *        height:number
  * }}
  */
-DropDownDiv.BoundsInfo;
+let BoundsInfo;
+exports.BoundsInfo = BoundsInfo;
```

Comment:
@cpcallen does this work as expected, or does the type annotation need to be on the export? I'm not quite sure how that all interacts.

Reply:
This is how [other type defs](https://github.com/google/blockly/blob/c9be8d984d6cc459917f2a777e03ba858f2e623d/core/positionable_helpers.js#L62) work I think. So hopefully should be good? If not I can go through and change those as well.

Reply:
I believe this is fine.  And it will soon be a single `export let …`, which is definitely fine as far as annotating it goes.